### PR TITLE
Use the value prop from v-model

### PR DESCRIPTION
**Pull request by francois2metz**
See #6 

The selected prop is not really needed.

### CODE REVIEW

**Comment by drewjbartlett** [comment]
I ran this on my local machine and I've noticed it now takes 2 initial clicks before the switch is changed. I'm assuming this may be potentially because of changing from @click to @input but I may be wrong. I don't have the time to dig into this at the moment but I really appreciate the help on this! I will definitely accept this PR once we get that issue figured out :)

**Comment by francois2metz** [comment]
The initial PR included the change to `@input`, but it was a mistake. I since fixed it.

I'll look at it.

**Comment by francois2metz** [comment]
I cannot reproduce the issue. Do you have a testcase?

**Comment by korzhyk** [comment]
Use `@change` instead of `@click`.

File: switches.vue [hunk 1]
```diff
@@ -6,7 +6,7 @@
             <span v-if="!label && !value" v-text="textDisabled"></span>
         </span>
 
-        <input type="checkbox" :disabled="disabled" @input="trigger">
+        <input type="checkbox" :disabled="disabled" @click="trigger">
```
Comment (korzhyk) [inline switches.vue]: Need to add `:checked="value"` for proper rendering actual state of checkbox.

**Review by korzhyk — CHANGES_REQUESTED** [review]
Use `@change` instead of `@click`.

**Comment by francois2metz** [comment]
Thanks for your review @korzhyk. I updated the code.

**Comment by drewjbartlett** [comment]
That fixed it! Thanks!

**Comment by kalashnikovisme** [comment]
I have to use `@input` in such case.
`@click` and `@change` don't work unfortunately :(

Version: 2.0.1

**Comment by kalashnikovisme** [comment]
What may be do I wrong?

**Pull request merged**